DuckDuckGo-macOS.xcodeproj/xcshareddata/xcschemes/macOS Browser App Store.xcscheme

```diff
@@ -239,6 +229,16 @@
                ReferencedContainer = "container:LocalPackages/HistoryView">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "SyncUITests"
+               BuildableName = "SyncUITests"
+               BlueprintName = "SyncUITests"
+               ReferencedContainer = "container:LocalPackages/SyncUI-macOS">
```

Comment:
This was a regression from the SyncUI rename, sorry 😢 Xcode didn't surface this in any way whatsoever.